CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 All notable changes to this project will be documented in this file.
 
+## Version 2.4
+
+- Param in fields section extended with `init: boolean`
```

Comment:
Need to mention about breaking with removed `data` section

Reply:
Need to describe new features and breaking changes separately under

### New

### Breaking changes 

sections